### PR TITLE
doc: edit AI Kit names and visibility in top-level README

### PR DESCRIPTION
**Pull request by dbkinder**
Signed-off-by: David B. Kinder <david.b.kinder@intel.com>

### CODE REVIEW

**Comment by dbkinder** [comment]
@jitendra42 Here's the PR to tweak AI Kit name and visibility...

File: README.md [hunk 1]
```diff
@@ -16,11 +18,19 @@ For any performance and/or benchmarking information on specific Intel platforms,
...
+- AI Kit provides a consolidated package of Intel’s latest deep and machine
+  learning optimizations all in one place for ease of development. Along with
+  Model Zoo,  the toolkit also includes Intel optimized versions of deep
+  learning frameworks (TensorFlow, PyTorch) and high performing Python libraries
+  to streamline end-to-end data science and AI workflows on Intel architectures.
+
+  |[Download AI Kit](https://software.intel.com/content/www/us/en/develop/tools/oneapi/ai-analytics-toolkit/) |[AI Kit Get Started Guide](https://software.intel.com/content/www/us/en/develop/documentation/get-started-with-ai-linux/top.html) |
+  |---|---|
+
+- To get started you can refer to [ResNet50 FP32 Inference code sample.](https://github.com/intel/AiKit-code-samples/tree/master/Intel_Model_Zoo_with_Tensorflow)
```
Comment (ashahba) [inline README.md:33]: This link is broken. Looks like the new repo is here: https://github.com/oneapi-src/oneAPI-samples but the desired sample is not there 🤔

File: README.md [hunk 2]
```diff
@@ -3,6 +3,8 @@
 This repository contains **links to pre-trained models, sample scripts, best practices, and step-by-step tutorials** for many popular open-source machine learning models optimized by Intel to run on Intel® Xeon® Scalable processors. 
 
 Model packages and containers for running the Model Zoo's workloads can be found at the [Intel® oneContainer Portal](https://software.intel.com/containers).
+Intel Model Zoo is also released as a part of
```
Comment (ashahba) [inline README.md:6]: Would this wording make more sense?
```
Intel Model Zoo is also bundled as a part of
```

**Review by ashahba — COMMENTED** [review]
Thanks @dbkinder for this PR.
Looks pretty good with a couple minor comments.

**Comment by dbkinder** [comment]
@ashahba Suggested edits made... thanks.

**Comment by dbkinder** [comment]
@ashahba Can this PR get merged (it removes the broken link you caught, to the sample that's not there anymore...)

**Review by ashahba — APPROVED** [review]
LGTM

**Pull request merged**